src/main/java/com/google/sps/server/ScheduleCalendar.java

```diff
@@ -74,12 +74,20 @@ public void addSchedule(Schedule schedule) throws IOException {
 
       // for each course in the schedule, add an event to the calendar
       for (ScheduledCourse currentCourse : schedule.getCourses()) {
-        List<TimeRange> sectionTimes = currentCourse.getSection().getMeetingTimes();
-        for (int i = 0; i < sectionTimes.size(); i++) {
-          String startTime = CalendarUtils.calculateStartTime(sectionTimes.get(i), month, year, nextSun);
-          String endTime = CalendarUtils.calculateEndTime(sectionTimes.get(i), month, year, nextSun);
+        List<TimeRange> lectureSectionTimes = currentCourse.getLectureSection().getMeetingTimes();
```

Comment:
Is `getLectureSection()` implemented in ScheduledCourse, or are you filling it in for now

Reply:
Maceo already added getLectureSection() to ScheduledCourse on the HandleRecitations branch

Reply:
Oh ok. I see it now, I completely missed it then

Reply:
No problem !